pages/asru/index.js

```diff
@@ -0,0 +1,83 @@
+const bodyParser = require('body-parser');
+const { page } = require('@asl/service/ui');
+const { buildModel } = require('@asl/pages/lib/utils');
+const form = require('@asl/pages/pages/common/routers/form');
+const { schema, getSchemaWithInspectors, getSchemaWithSpocs } = require('./schema');
+
+module.exports = settings => {
+  const app = page({
+    root: __dirname,
+    ...settings
+  });
+
+  app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
the form router attached the bodyParser middleware on POST - do we need it here too? Also the form uses `extended: false` to allow string/array values - is there a reason it is true here?

Reply:
removed it.

Reply:
actually, it does not seem to be case ... I mean , I can see the attachment in https://github.com/UKHomeOffice/asl-pages/blob/e676092c4a6f01dc48ad63e534ecd06e5f9c2f6d/pages/common/routers/form.js#L172 , but if I remove app.use(bodyParser.urlencoded({ extended: false })); parameters are not found in req.body ...

Reply:
aah - that's because we are parsing the body on `POST /` but not `POST /delete`. In that case I would add the middleware in the post handler that needs it - 

```js
app.post('/delete', bodyParser.urlencoded({ extended: false }), (req, res, next) => {
  // your code
})
```

Reply:
TBH, I'd probably move body-parsing to a global level. There's basically never going to be a context in which we're doing a POST and _don't_ want to parse the body.

Reply:
done